ubuntu-advantage.1

```diff
@@ -1,157 +1,130 @@
 
-.TP
-.B
-version
-Show version.
+This shows whether this machine is attached to an Ubuntu Advantage
+support contract. When attached, the report includes the specific
+support contract details including contract name, expiry dates, and the
+status of each service on this system.
 
-.SH ESM (Extended Security Maintenance)
-Ubuntu Extended Security Maintenance archive. See https://ubuntu.com/esm for
-more information.
-.TP
-Enabling the ESM repository provides access to ongoing security fixes for
-end-of-life Ubuntu LTS releases.
+Each service status line has three columns:
 
-.SH FIPS (Canonical FIPS 140-2 certified modules)
-Install, configure, and enable FIPS 140-2 certified modules.
-.TP
-Enabling the FIPS PPA repository, installs the FIPS modules, configures
-the bootloader and enables fips on the system. After executing
-"ubuntu-advantage enable fips", the system MUST be rebooted to
-complete the enablement process. Failing to reboot will result in the system
-not being fips enabled.
+.BR "name" ":"
+name of the service
 
-The following FIPS certified modules will be installed and put in fips mode;
-openssh-server, openssh-client, strongswan, openssl, and the kernel
-cryptoapi.
+.BR "availability" ":"
+whether the contract entitles use of this service. Possible values are:
+none or entitled
 
-Disabling FIPS is not currently supported.
+.BR "status" ":"
+whether the service is activated on this machine.
+Possible values are: active, inactive, or n/a
 
-.SH FIPS (Canonical FIPS 140-2 certified modules with updates)
-Install, configure, and enable FIPS 140-2 certified modules with updates
-.TP
-.B
-Enabling the FIPS-updates entitlement with "ua enable fips-updates" will take
-the system out of FIPS compliance as the updated modules are not FIPS
-certified. The option enables the FIPS-UPDATES PPA repository and installs the
-updated FIPS modules. If the system is installing FIPS modules for the first
-time, it configures FIPS on the system. After successfully executing the
-ubuntu-advantage to update FIPS modules, the system MUST be rebooted if FIPS
-kernel was upgraded in the upgrade process. Failing to reboot will result
-in the system not running the updated FIPS kernel.
-
-The following FIPS modules will be updated and put in FIPS mode -
-openssh-server, openssh-client, strongswan, openssl, and the kernel
-cryptoapi.
-
-.SH Livepatch (Canonical Livepatch Service)
-Managed live kernel patching. For more information, visit
-https://www.ubuntu.com/server/livepatch
 .TP
-Enable the Livepatch service on this machine to allow critical kernel patches
-without requiring a reboot.
+.B version
+Show version of the Ubuntu Advantage package.
 
-.SH EXIT STATUS
-.TP
-.B
-0
-Command succeded
-.TP
-.B
-1
-Invalid command or option
-.TP
-.B
-2
-Current user is not root
-.TP
-.B
-3
-Invalid or missing token when enabling a service
 .TP
-.B
-4
-The requested service is not supported on the current Ubuntu release
+.B detach
```

Comment:
This is the only command not in its alphabetical place; was this intentional?

Reply:
not intentional, I can't spell...